### PR TITLE
Fix misc stuff from lighthouse

### PR DESCRIPTION
**Pull request by TomNUSDS**
- Icons in list were resizing to be too small.
- Added robots.txt (full exclude for now)
- Add favicon.ico placeholder
- Added page title using helmet (react-helmet + gatsby-plugin-react-helmet)
Note: not sure if react-helmet can be in dev section of package.json?

### CODE REVIEW

File: client/static/robots.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Disallow: /
```
Comment (NatHillardUSDS) [inline client/static/robots.txt:2]: Makes sense for now; we'll need to readdress closer to launch -- might be worth a ticket?

File: client/package.json [hunk 2]
```diff
@@ -50,9 +51,11 @@
   },
   "dependencies": {
     "@trussworks/react-uswds": "github:nathillardusds/react-uswds#nathillardusds/ssr",
+    "@types/react-helmet": "^6.1.1",
     "gatsby": "^3.4.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-helmet": "^6.1.0",
```
Comment (NatHillardUSDS) [inline client/package.json:58]: as part of the security audit PR we're going to move these around; we can leave as-is for now

**Review by NatHillardUSDS — APPROVED** [review]
LGTM!

File: client/static/robots.txt (hunk 1 above)
Comment (TomNUSDS) [reply client/static/robots.txt:2]: https://github.com/usds/justice40-tool/issues/84

File: client/package.json (hunk 2 above)
Comment (TomNUSDS) [reply client/package.json:58]: Yeah, wasn't 100% sure where it needed to be. I'll move it into dev section and test.

**Pull request merged**